src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -817,3 +817,7 @@ Prevent Long URL From Breaking Out of Container
         margin: 10px;
     }
 }
+
+.student_remind_tick {
+    width: 15px;
+}
```

Comment:
Our naming convention for CSS classes is to join the terms with hyphens.

The name can also be more generic in terms of where it can be used (maybe remove the `student`?) and more specific in terms of where it should be used (maybe add a `table column`?). `check` or `checkbox` would be more explanatory than `tick` too.

Centering the checkbox would allow it to look better, try using padding or margin instead of a fixed width.

Lastly, as this has nothing to do with responsiveness, I don't believe this rule should belong on these lines.